testing/MLDB-1012_nested_function_calls.py

```diff
@@ -79,14 +79,14 @@
 js_rez = rez.json()
 assert js_rez[0]['columns'][0][1] == 1
 
-mldb.put("/v1/functions/recurse", {
-    "type": "sql.expression",
-    "params": {"expression": "recurse({input})[output] as output"}
-})
-
 # MLDB-1251
 mldb.log("MLDB-1251")
 try:
+    mldb.put("/v1/functions/recurse", {
+        "type": "sql.expression",
+        "params": {"expression": "recurse({input})[output] as output"}
+    })
```

Comment:
The test actually did check that the binding didn't cause an infinite loop; it's just that the binding now happens earlier than before, and so the test didn't test where the infinite loop was happening anymore.  It's testing that the same exception was thrown from the same function, so it's basically the same as before.